app/views/books/_translations.html.erb

```diff
@@ -47,6 +47,6 @@ This book is available in
 </p>
 <hr class="sidebar"/>
 <p>
-The source of this book is <a href="https://github.com/progit/progit2">hosted on GitHub.</a></br>
+The source of this book is <%if @book.code == "en" %><a href="https://github.com/progit/progit2">hosted on GitHub.</a><% else %> <a href="https://github.com/<%= Book.all_books[@book.code] %>">hosted on GitHub.</a><% end %></br>
```

Comment:
Nice. I'm on the fence on whether translations should mention the main repo (e.g., for technical fixes). I guess we can leave that up to the README in the individual translation repos.

Reply:
Not sure about that. For French, we had mostly reports for translation improvements. When reporting, you can never know whether the mistake is in the main repo or at the translation level.

Reply:
OK. Let's go with this, then. It's certainly an improvement over the existing code, and we can always revisit it later if we see a lot of misdirected issues (but you'd know about that much more than me).